src/test/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAssociationManagerTest.java

```diff
@@ -66,11 +66,12 @@ public final void setUp() throws Exception {
   /**
    * Verify that the constructor responds correctly to bad input.
    *
-   * @throws SegueDatabaseException
-   * @throws UserGroupNotFoundException
+   * @throws SegueDatabaseException test exception
+   * @throws UserGroupNotFoundException test exception
    */
+  @SuppressWarnings("checkstyle:NonEmptyAtclauseDescription")
```

Comment:
Please don't suppress warnings. Instead, fix whatever needs fixing, or ask a question in the Slack channel if you're unsure. This one in particular is not necessary and should be removed.

Reply:
I think that was done buy accident - I have removed it and will be pushed up with any more updated changes.

Reply:
Perfect, thank you!!